packit_service/worker/handlers/testing_farm.py

```diff
@@ -116,6 +120,44 @@ def get_checkers() -> Tuple[Type[Checker], ...]:
             CanActorRunJob,
         )
 
+    def _get_or_create_group(
+        self,
+        targets: List[str],
+        copr_build: Optional[CoprBuildTargetModel],
+        copr_builds_for_targets: List[CoprBuildTargetModel],
+    ) -> TFTTestRunGroupModel:
```

Comment:
could we please have a docstring here since from the first look it may not be clear what the arguments really are and generally how is the creation of the group done?